cmd/vorteil/main.go

```diff
@@ -121,6 +122,38 @@ const (
 	sourceINVALID            = "INVALID"
 )
 
+func readSourcePath(src string) (string, string, error) {
+
+	src, err := filepath.Abs(src)
+	if err != nil {
+		return "", "", err
+	}
+
+	// Check if source contains a target
+	colonSplit := strings.Split(src, ":")
+	var target string
+
+	// colonSplit will always be 1 when strings.splitting something that doesn't have it
+	if len(colonSplit) > 1 {
```

Comment:
maybe check with LastIndex and cut string there? It is either "not found" or something legit.

Reply:
I'm now setting target to the last of the split array. A few other if statments exist like

Set target empty if its the same as src. Happens on linux
```
if target == src {
 target = ""
}
```
And the following for windows. If windows doesn't have a length of 3 then the target is not provided 
```
if colonLength == 2 {
target = ""
}
```